### PR TITLE
Fix workspace is undefined bug

### PR DESCRIPTION
**Pull request by dvargasfuertes**
Screenshot 2024-06-01 at 6 32 42 AM

### CODE REVIEW

File: api/oauth/post.ts [hunk 1]
```diff
@@ -65,12 +74,20 @@ export const logic = async (inputArgs: {
     installationId: "",
   };
   if (args.grant_type === "authorization_code") {
-    const { email, localId, displayName } = await axios
+    const {
+      email,
+      localId = email,
```
Comment (dvargasfuertes) [inline api/oauth/post.ts:79]: damn, the docs claim that this has a value...

File: api/oauth/post.ts [hunk 2]
```diff
@@ -65,12 +74,20 @@ export const logic = async (inputArgs: {
...
+      displayName,
+    } = await axios
       // https://cloud.google.com/identity-platform/docs/reference/rest/v1/UserInfo
       .get<{ email: string; localId: string; displayName: string }>(
         `https://www.googleapis.com/oauth2/v2/userinfo?access_token=${data.access_token}`
       )
-      .then((u) => u.data)
+      .then((u) => {
+        console.log("Found User Info:");
```
Comment (dvargasfuertes) [inline api/oauth/post.ts:86]: this logging need is increasing for me

File: api/oauth/post.ts (hunk 1 above)
Comment (dvargasfuertes) [reply api/oauth/post.ts:79]: oh the docs is v1... and I can't find v2 -.-

**Review by mdroidian — COMMENTED** [review]
✅

**Pull request merged**